spark/dl/src/main/scala/com/intel/analytics/bigdl/example/loadmodel/DatasetUtil.scala

```diff
@@ -32,24 +33,35 @@ object AlexNetPreprocessor {
 
   def apply(path: String, batchSize: Int, meanFile: String, sc: SparkContext)
   : DataSet[MiniBatch[Float]] = {
-    val means = File.load[Tensor[Float]](meanFile)
+    val means = createMeans(meanFile)
```

Comment:
Add comments to explain the format of meanFile? And may need provide a download link to get the meanFile in the readme

Reply:
Comments added, User should have option to convert from imagenet mean file as long as we provide the format, we could upload a preconverted one as well